Dockerfile

```diff
@@ -9,20 +9,23 @@ ENV DISPLAY :99
 COPY test/xvfb_init /etc/init.d/xvfb
 COPY test/xvfb_daemon_run /usr/bin/xvfb-daemon-run
 
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-    echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-    apt-get update -yy -qq && \
-    apt-get install yarn -yy -qq && \
-    apt-get install xvfb firefox-esr -yy -qq && \
-    chmod a+x /etc/init.d/xvfb /usr/bin/xvfb-daemon-run
-
 COPY yarn.lock package.json ./
 
 RUN mkdir -p docs test packages/gestalt
 COPY packages/gestalt/package.json ./packages/gestalt/
 COPY docs/package.json ./docs/
 COPY test/package.json ./test/
 
+RUN  apt-get update \
```

Comment:
I think you want this `RUN` command to be high so it's not invalidated by changes to the `package.json` etc.